scipy/optimize/tests/test__differential_evolution.py

```diff
@@ -714,16 +715,19 @@ def test_immediate_updating(self):
     def test_parallel(self):
         # smoke test for parallelization with deferred updating
         bounds = [(0., 2.), (0., 2.)]
-        with multiprocessing.Pool(2) as p, DifferentialEvolutionSolver(
-                rosen, bounds, updating='deferred', workers=p.map) as solver:
-            assert_(solver._mapwrapper.pool is not None)
-            assert_(solver._updating == 'deferred')
+        # use threads instead of Process to speed things up for this simple example
```

Comment:
I think it's harder for CI agents to create Processes. It's reasonably fast locally.